<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QLabel" name="TitleDialog">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter the data</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="BtnApply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="BtnCancel">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Lblmnemonic_id">
       <property name="text">
        <string>mnemonic_id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lblmnemonic">
       <property name="text">
        <string>mnemonic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lblunit">
       <property name="text">
        <string>unit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lbldescription">
       <property name="text">
        <string>description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lbltypedevice_id">
       <property name="text">
        <string>typedevice_id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Lblparent_mnemonic_id">
       <property name="text">
        <string>parent_mnemonic_id</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lbmnemonic_id">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lbmnemonic">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lbunit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lbdescription">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lbtypedevice_id">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lbparent_mnemonic_id">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
